rhoas-client/src/main/java/com/redhat/service/bridge/rhoas/dto/TopicAndServiceAccountRequest.java

```diff
@@ -0,0 +1,42 @@
+package com.redhat.service.bridge.rhoas.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
@kostola I think I need to see where the `RhoasClient` is used in response to creating an OpenBridge Instance to offer feedback on this. My initial reaction is that the `RhoasClient` should take care of generating the topic name and service account name and not the thing calling it.

Reply:
I integrated `RhoasClient` in the manager inside `BridgesServiceImpl`. Take a look :nerd_face: